### PR TITLE
Big scrape: no casefolding

### PR DESCRIPTION
**Pull request by kylebgorman**
This responds to #469 and #471 by removing casefolding from the big scrape. If accepted, the big scrape will also be re-run with this in place.

- [x] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

### CODE REVIEW

File: data/scrape/lib/generate_summary.py
Comment (jacksonllee) [inline data/scrape/lib/generate_summary.py:1]: I pulled down this branch and ran the `generate_summary.py` script to verify your changes. The resulting `data/scrape/README.md` looked slightly off, which would require a few minor tweaks in `generate_summary.py`:

* Drop `"case_folding"`: https://github.com/CUNY-CL/wikipron/blob/b122b18c37df81321207e8cac14a95c3a0827852/data/scrape/lib/generate_summary.py#L109
* Drop `Case-folding | `: https://github.com/CUNY-CL/wikipron/blob/b122b18c37df81321207e8cac14a95c3a0827852/data/scrape/lib/generate_summary.py#L145
* Change 8 into 7: https://github.com/CUNY-CL/wikipron/blob/b122b18c37df81321207e8cac14a95c3a0827852/data/scrape/lib/generate_summary.py#L149

**Review by jacksonllee — COMMENTED** [review]
`generate_summary.py` would need a few small updates, but otherwise this pull request looks good!

File: data/scrape/lib/generate_summary.py
Comment (kylebgorman) [reply data/scrape/lib/generate_summary.py:1]: I thought I'd fixed this already, must have forgot to stage! Fixed. I just ran `generate_summary.py` and inspected the output---it renders correctly now.

**Review by jacksonllee — APPROVED** [review]
LGTM!

**Pull request merged**